docs/docs-content/vertex/system-management/account-management/password-blocklist.md

```diff
@@ -0,0 +1,259 @@
+   blocklist.
+
+## Remove Passwords from the Blocklist
+
+Use the following steps to remove passwords from the password blocklist.
+
+### Prerequisites
+
+- Access to the Palette VerteX system console.
+
+- The role of root administrator or operations administrator.
+
+- curl or any other tool to make HTTP requests.
+
+- Access to a shell terminal.
+
+:::warning
+
+If you enabled passkeys for your system administrator account, the System API login endpoint will be disabled. Refer to
+the [Passkeys and API Access](./credentials.md#passkeys-and-api-access) for guidance on how to use passkeys to access
+the System API.
+
+:::
+
+### Remove Passwords
+
+1.  Open a terminal session and log in to the Palette VerteX System API by using the `/v1/auth/syslogin` endpoint. Use
+    the `curl` command below and replace the URL with the custom domain URL you assigned to Palette VerteX or use the IP
+    address. Ensure you replace the credentials below with your system console credentials.
+
+    ```bash
+    curl --location 'https://vertex.example.com/v1/auth/syslogin' \
+    --header 'Content-Type: application/json' \
+    --data '{
+      "password": "**********",
+      "username": "**********"
+    }'
+    ```
+
+    :::tip
+
+    If you are using the default self-signed certificate, you can use the `--insecure` flag to bypass the certificate
+    check.
+
+    :::
+
+    Output
+
+    ```json hideClipboard
+    {
+      "Authorization": "**********.",
+      "IsPasswordReset": true
+    }
+    ```
+
+2.  Using the output you received, copy the authorization value to your clipboard and assign it to a shell variable.
+    Replace the authorization value below with the value from the output.
+
+    ```shell hideClipboard
+    TOKEN=**********
+    ```
+
+3.  Use the `curl` command below to insert new passwords into the password blocklist. Replace the URL with the custom
+    domain URL you assigned to Palette VerteX or use the IP address. Ensure you replace the `TOKEN` value with the value
+    you assigned to the shell variable.
+
+    ```bash
+    curl --location --request DELETE 'https://vertex.example.com/v1/sys/passwords' \
+    --header "Authorization: $TOKEN" \
+    --header 'Content-Type: application/json' \
+    --data '{
+      "spec": {
+          "passwords": [
+                "Password",
+                "password",
+                "123456",
+                "qwerty",
+          ]
+      }
+    }'
+    ```
+
+    :::info
+
+    The payload expects the `spec.passwords`field to be an array of strings. You can add as many passwords as you want
+    as long as they are separated by a comma.
+
+    :::
+
+4.  The endpoint returns a `204` status code if the operation is successful. No output is returned. If the operation
+    fails, the endpoint returns an error message. In case of an error, verify the authorization token is valid and the
+    password blocklist is in the expected format.
```

Comment:
Does the user need to make sure the password they want to remove is in the blocklist?